src/pages/Editor.vue

```diff
@@ -368,6 +368,7 @@
             </div>
             <!--- item editor  -->
             <item-editor
+              :key="componentKey"
```

Comment:
this key is specific to the purpose of re-rendering right?
Better to keep consistency and do what we've done in rest of the codebase, named it `reRenderKey`

Reply:
I chose this name because there was already a key with the name `reRenderKey`, but yess,it should still work. I'll have another look at the code.